examples/pytorch/ogb/ngnn_seal/main.py

```diff
@@ -505,8 +508,12 @@ def print_log(*x, sep="\n", end="\n", mode="a"):
 
     if 0 < args.sortpool_k <= 1:  # Transform percentile to number.
         if args.dynamic_train:
-            _sampled_indices = range(1000)
-            #_sampled_indices = np.random.choice(
+            _sampled_indices = list(range(1000))
+            if args.dataset.startswith("ogbl-citation"):
+                _sampled_indices = _sampled_indices + list(
+                    range(len(train_dataset) - 1000, len(train_dataset))
+                )
```

Comment:
Why don't you directly do something like

```
if args.dataset.startswith("ogbl-citation"):
    _sampled_indices = list(range(len(train_dataset)))
else:
    _sampled_indices = list(range(1000))
```

The code will be less convoluted.